app/assets/stylesheets/petitions/views/_petition-show.scss

```diff
@@ -8,14 +8,92 @@
   }
 }
 
+
+// Custom animation
+@-webkit-keyframes fadeInLeft {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    transform: translate3d(-100%, 0, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+@keyframes fadeInLeft {
+  0% {
+    opacity: 0;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    transform: translate3d(-100%, 0, 0);
+  }
+
+  100% {
+    opacity: 1;
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+%animated {
```

Comment:
@pixeltrix @alexander-bobin – the animation only adds about 10 lines to the output css.
`%extend` syntax in scss isn't compiled unless called in a class. For example:
```.warning {
  @extend %animated;
  @extend %bounce;
}```
Thought this was a pragmatic balance but happy to just copy and paste the lines used into place.